jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/LocalBaseImageStepsTest.java

```diff
@@ -67,22 +67,24 @@ public void setup() throws IOException {
     Mockito.when(progressEventDispatcher.newChildProducer()).thenReturn(childFactory);
     Mockito.when(childFactory.create(Mockito.anyString(), Mockito.anyLong()))
         .thenReturn(childDispatcher);
+    tempDirectoryProvider = new TempDirectoryProvider();
```

Comment:
I don't see any problem with

`private *final* TempDirectoryProvider tempDirectoryProvider = new TempDirectoryProvider();`

Reply:
Since we're not passing a `TempDirectoryProvider` anymore, looks like we can just remove this.